workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/WorkerDetail.cs

```diff
@@ -0,0 +1,112 @@
+using System.Collections.Generic;
+using Improbable.Gdk.Core;
+using Unity.Entities;
+using UnityEditor;
+using UnityEngine.UIElements;
+
+namespace Improbable.Gdk.Debug.WorkerInspector
+{
+    internal class WorkerDetail : VisualElement
+    {
+        private const string FlagKeyName = "key";
+        private const string FlagValueName = "value";
+
+        private readonly TextField workerType;
+        private readonly TextField workerId;
+        private readonly ListView workerFlags;
+
+        private readonly List<KeyValuePair<string, string>> workerFlagData = new List<KeyValuePair<string, string>>();
+
+        private View view;
+
+        public WorkerDetail()
+        {
+            const string uxmlPath = "Packages/io.improbable.gdk.debug/WorkerInspector/Templates/WorkerDetail.uxml";
+            var template = AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(uxmlPath);
+            template.CloneTree(this);
+
+            workerType = this.Q<TextField>("worker-type");
+            workerType.SetEnabled(false);
+
+            workerId = this.Q<TextField>("worker-id");
+            workerId.SetEnabled(false);
+
+            workerFlags = this.Q<ListView>("worker-flags-list");
+            workerFlags.makeItem = MakeItem;
+            workerFlags.bindItem = BindElement;
+            workerFlags.itemsSource = workerFlagData;
+            workerFlags.SetEnabled(false);
+        }
+
+        public void SetWorld(World world)
+        {
+            if (world == null)
+            {
+                workerType.value = "";
+                workerId.value = "";
+                workerFlagData.Clear();
+                workerFlags.Refresh();
+                view = null;
+                return;
+            }
+
+            var workerSystem = world.GetExistingSystem<WorkerSystem>();
+
+            workerType.value = workerSystem.WorkerType;
+            workerId.value = workerSystem.WorkerId;
+
+            view = workerSystem.View;
+            Update();
+        }
+
+        public void Update()
+        {
+            if (view == null)
+            {
+                return;
+            }
+
+            workerFlagData.Clear();
+
+            foreach (var pair in view.WorkerFlags)
+            {
+                workerFlagData.Add(pair);
+            }
+
+            workerFlags.Refresh();
+        }
+
+        private static VisualElement MakeItem()
+        {
+            var container = new VisualElement();
+            container.AddToClassList("flag-element-container");
+
+            var key = new Label { name = FlagKeyName, };
+
```

Comment:
Why all the whitespace here?

Reply:
The initializer was spanning multiple lines, but then I linted and it collapsed it. 🤷‍♂️